integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/SparkMockEntity.java

```diff
@@ -0,0 +1,112 @@
+package com.sequenceiq.it.cloudbreak.newway;
+
+import static com.sequenceiq.it.cloudbreak.newway.Mock.createTempFileFromClasspath;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Stack;
+
+import com.sequenceiq.it.cloudbreak.newway.mock.MockModel;
+import com.sequenceiq.it.verification.Call;
+
+import spark.Response;
+import spark.Service;
+
+public class SparkMockEntity extends Entity {
```

Comment:
This could also be removed as the MockServer is part of the DefaultModel/Testcontext.

Reply:
These classes are used in the legacy code.